main.js

```diff
@@ -0,0 +1,18 @@
+var tableElement = document.querySelector('.table')
+var headerElement = document.querySelector('.header')
+var formElement = document.querySelector('form')
+var noGradesElement = document.querySelector('.no-grades')
+var updateElement = document.querySelector('.update-form')
+var dataArray = []
+
+var gradeForm = new GradeForm(formElement)
+
+var pageHeader = new PageHeader(headerElement)
+
+var gradeTable = new GradeTable(tableElement, noGradesElement)
+
+var updateGradeForm = new UpdateGradeForm(updateElement)
```

Comment:
Adding a second form component (`UpdateGradeForm`) is a great opportunity to give the first one a more specific name (`CreateGradeForm` or `AddGradeForm`, instead of `GradeForm`).